base/array.jl

```diff
@@ -1080,6 +1080,8 @@ end
 Remove an item in `collection` and return it. If `collection` is an
 ordered container, the last item is returned.
 
+See also: [`shift!`](@ref), [`splice!`](@ref)
```

Comment:
What about `push!`

Reply:
Push! adds elements, whereas the other functions remove elements. I think, `See also:`-lists should be rather minimal: They reduce the need to search the documentation but it's no replacement for a proper search.

Reply:
A thesaurus lists both similar words and antonyms for a reason – knowing about `push!` is useful if you're looking at `pop!`. From `push!` you could potentially find all the other element addition functions.